packages/amplify_authenticator/lib/src/state/auth_viewmodel.dart

```diff
@@ -337,15 +337,10 @@ class AuthViewModel extends ChangeNotifier {
   }
 
   Future<void> resendSignUpCode() async {
-    if (!_formKey.currentState!.validate()) {
```

Comment:
Since resending the code doesn't submit the form, we shouldn't care if the form is valid.